docs-source/content/userguide/introduction/introduction.md

```diff
@@ -19,7 +19,8 @@ Detailed instructions are available [here]({{< relref "/userguide/managing-opera
 For the current production release 3.0.2:
 
 * Kubernetes 1.14.8+, 1.15.7+, 1.16.0+, 1.17.0+, and 1.18.0+ (check with `kubectl version`).
-* Flannel networking v0.9.1-amd64 or later (check with `docker images | grep flannel`) *or* OpenShift SDN on OpenShift 4.3 systems.
+* Flannel networking v0.9.1-amd64 or later (check with `docker images | grep flannel`), Calico networking (Calico v3.16.1 or Calico Enterprise v3.2.0),
```

Comment:
We should remove reference to calico Enterprise. The kind cluster script we use only installs calico and not the enterprise version and so we did not test it. I have asked Mark if we should test specifically with the enterprise version and awaiting his response.

Reply:
Removed Calico Enterprise v3.2.0.